skins/oasis/css/modules/ContributeModule.scss

```diff
@@ -0,0 +1,46 @@
+@import 'skins/shared/mixins/flexbox';
+
+.contribute-module {
```

Comment:
Fixed in [63aa6f5](https://github.com/Wikia/app/commit/63aa6f5d3ba86c91b0bf043d6c0c1011ac41892e)